### ISSUE TITLE
[Bug] Walk on water not working in rivers?

### ISSUE THREAD

**Issue by TheRealAyCe** [gta-chaos-mod/ChaosModV#1038]
https://www.twitch.tv/videos/984800744?t=0h28m0s
0:28:00

I guess it only works for the ocean?

**Comment by anto4440** [comment]
Nope, it didnt work because he was already in the water.

**Comment by Last0xygen** [comment]
> Nope, it didnt work because he was already in the water.

Thats not true. Walk on water teleports you on top of the invisible barrier if you are below it.
Seems like the native function that should return the water level does not correctly return the water level on those rivers

**Comment by anto4440** [comment]
I see.
